plutus-benchmark/nofib/test/9.6/clausify-F5.size.golden

```diff
@@ -1 +1 @@
-1598
\ No newline at end of file
+1615
```

Comment:
Does this test run with `inline-threshold` set to something else other than the default?